application/src/Api/Representation/AbstractResourceEntityRepresentation.php

```diff
@@ -568,6 +594,7 @@ public function displayResourceClassLabel($default = null)
      * @return string
      */
     public function linkPretty(
+        $lang = null,
```

Comment:
Adding `lang` as the first parameter to linkPretty messes with the signature which we probably don't want to do.

Reply:
You're totally right. Updated in 81ec357